java/build.xml

```diff
@@ -0,0 +1,56 @@
+<!--
+ Ant build file for the project jar.
+
+ All functionality comes from the common-module.xml Ant script.
+ Do not add targets or properties here unless they are
+ only specific to this module.
+
+ Note that including the local ant.properties file must
+ happen before importing the common-module.xml to ensure
+ properties are overridden correctly.
+
+ Type "ant -p" to see the list of available targets.
+-->
+<project name="tab-documentation-api" basedir="." default="main" xmlns:ivy="antlib:org.apache.ivy.ant">
```

Comment:
+1 I immediately thought this was a tool for writing Tableau documentation or something.